webview/src/stories/Ribbon.stories.tsx

```diff
@@ -0,0 +1,61 @@
+import React from 'react'
```

Comment:
[F] Add a story to demonstrate the styling of the ribbon with and without progress ring(s)

Reply:
I'm sure we'll need to iterate on the style.